spacy_llm/backends/integration/hf/__init__.py

```diff
@@ -1,9 +1,11 @@
 from .base import HuggingFaceBackend
 from .dolly import backend_dolly_hf
+from .openllama import backend_openllama_hf
 from .stablelm import backend_stablelm_hf
 
 __all__ = [
     "HuggingFaceBackend",
     "backend_dolly_hf",
     "backend_stablelm_hf",
+    "backend_openllama_hf",
```

Comment:
As a general comment, let's keep these alphabetized to facilitate maintainability.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/151/commits/2732d5993f7afd002270d036731d07d9c0504a22.